sources/Core/Numerics/Quaternion.cs

```diff
@@ -21,74 +23,166 @@ namespace TerraFX.Numerics
+        public static Quaternion operator +(Quaternion left, Quaternion right) => new Quaternion(left._value + right._value);
+
+        /// <summary>Subtracts two specified <see cref="Quaternion" /> values.</summary>
+        /// <param name="left">The minuend.</param>
+        /// <param name="right">The subtrahend.</param>
+        /// <returns>The result of subtracting <paramref name="right" /> from <paramref name="left" />.</returns>
+        public static Quaternion operator -(Quaternion left, Quaternion right) => new Quaternion(left._value - right._value);
+
+        /// <summary>Multiplies two specified <see cref="Quaternion" /> values.</summary>
+        /// <param name="left">The first value to multiply.</param>
+        /// <param name="right">The second value to multiply.</param>
+        /// <returns>The result of multiplying <paramref name="left" /> by <paramref name="right" />.</returns>
+        public static Quaternion operator *(Quaternion left, Quaternion right)
         {
-            return (left.X != right.X)
-                || (left.Y != right.Y)
-                || (left.Z != right.Z)
-                || (left.W != right.W);
+            // from XMQuaternionMultiply at https://github.com/microsoft/DirectXMath/blob/master/Inc/DirectXMathMisc.inl#L82
+            // (r.x * +l.w) + (r.y * +l.z) + (r.z * -l.y) + (r.w * +l.x),
+            // (r.x * -l.z) + (r.y * +l.w) + (r.z * +l.x) + (r.w * +l.y),
+            // (r.x * +l.y) + (r.y * -l.x) + (r.z * +l.w) + (r.w * +l.z),
+            // (r.x * -l.x) + (r.y * -l.y) + (r.z * -l.z) + (r.w * +l.w)
+
+            var resultQ = new Quaternion(
+                Quaternion.Dot(left, new Quaternion(+right.W, +right.Z, -right.Y, +right.X)),
+                Quaternion.Dot(left, new Quaternion(-right.Z, +right.W, +right.X, +right.Y)),
+                Quaternion.Dot(left, new Quaternion(+right.Y, -right.X, +right.W, +right.Z)),
+                Quaternion.Dot(left, new Quaternion(-right.X, -right.Y, -right.Z, +right.W))
+            );
+            return resultQ;
         }
 
-        /// <summary>Creates a new <see cref="Quaternion" /> instance with <see cref="X" /> set to the specified value.</summary>
-        /// <param name="value">The new value of the x-component.</param>
-        /// <returns>A new <see cref="Quaternion" /> instance with <see cref="X" /> set to <paramref name="value" />.</returns>
-        public Quaternion WithX(float value) => new Quaternion(value, Y, Z, W);
+    /// <summary>Divides two specified <see cref="Quaternion" /> values component by component.</summary>
+    /// <param name="left">The dividend.</param>
+    /// <param name="right">The divisor.</param>
+    /// <returns>The result of dividing <paramref name="left" /> by <paramref name="right" />.</returns>
+    public static Quaternion operator /(Quaternion left, Quaternion right) => new Quaternion(left._value / right._value);
```

Comment:
This got misindented:
```suggestion
        /// <summary>Divides two specified <see cref="Quaternion" /> values component by component.</summary>
        /// <param name="left">The dividend.</param>
        /// <param name="right">The divisor.</param>
        /// <returns>The result of dividing <paramref name="left" /> by <paramref name="right" />.</returns>
        public static Quaternion operator /(Quaternion left, Quaternion right) => new Quaternion(left._value / right._value);
```